README.md

```diff
@@ -9,36 +9,50 @@ A juju charm deploying and managing Discourse on Kubernetes. Discourse is the
 100% open source discussion platform. Use it as a mailing list, discussion
 forum or long-form chat room.
 
-This charm simplifies initial deployment and "day N" operations of Discourse
-on Kubernetes, such as scaling the number of instances, integration with SSO,
-access to S3 for redundant file storage and more. It allows for deployment on
-many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) or
+Like any Juju charm, this charm supports one-line deployment, configuration, integration, scaling, and more.
+For Charmed Discourse, this includes:
+  - Scaling
+  - Integration with SSO
+  - Integration with S3 for redundant file storage
+
+It allows for deployment on
+many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
 [Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
 offerings.
 
-As such, the charm makes it easy for those looking to take control of their own
-discussion platform whilst keeping operations simple, and gives them the
-freedom to deploy on the Kubernetes platform of their choice.
+For information about how to deploy, integrate, and manage this charm,
+see the Official [Discourse Operator Documentation](https://charmhub.io/discourse-k8s/docs).
 
-For DevOps or SRE teams this charm will make operating Discourse simple and
-straightforward through Juju's clean interface. It will allow easy deployment
-into multiple environments for testing of changes, and supports scaling out for
-enterprise deployments.
+## Get started
 
-## Project and community
+You can follow the tutorial [here](https://charmhub.io/discourse-k8s/docs/tutorial).
+
+### Basic operations
+
+The following actions are available for this charm:
+    - anonymize-user: anonymize a user
+    - create-user: create a new user
+    - promote-user: promote a user to admin
```

Comment:
These lines aren't rendering as a bulleted list for me -- maybe because of how they're indented?

Another nitpick: I think it may look better to format the actions as code, in other words:

- `anonymize-user`: anonymize a user
- `create-user`: create a new user
- `promote-user`: promote a user to admin